src/main/target/OMNIBUSF7/target.h

```diff
@@ -168,6 +171,7 @@
 #define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
 #define SERIALRX_UART           SERIAL_PORT_USART2
 #define SERIALRX_PROVIDER       SERIALRX_SBUS
+#define ESC_SENSOR_UART         SERIAL_PORT_USART7
```

Comment:
Not really. All of the `<pg>Mutable()-><paramerter> = <value>` stanzas in `config.c` can simply be added as `set <pg>_<parameter> = <value>` to the config file.